Documentation: a draft about doc structure and style (#28)
aiidateam/kiwipy: docs/source/concepts.rst

.. _concepts:

> Here I preparing to firstly introduce the differences between task, rpc, and broadcast and their usage scenario, because the task in kiwipy is slightly different from the task defined in RabbitMQ's tutorial. The task in kiwipy is bidirectional and the return value can be obtained in client, which is more like a simple rpc. Thus easy to be confused with rpc mode.

Reviewer: ```suggestion
Here we introduce the meaning of task, rpc, and broadcast and their usage scenarios, since the meaning of a the task in ``kiwipy`` differs slightly from the one in RabbitMQ's tutorial.
```

I suggest to move the sentence "The task in kiwipy is bidirectional and the return value can be obtained by the client, which is more like a simple rpc. Thus easy to be confused with rpc mode." under the "Task" section

> Secondly, I prepare to introduce the concepts of loop dependent communicator and thread dependent communicator, which are objects might be used by other library directly.

Reviewer: ```suggestion
Furthermore, we introduce the concepts of loop-dependent and thread-dependent communicators, which are objects that might be used directly by dependent libraries.
```
